core/qt-ble.cpp

```diff
@@ -106,6 +109,11 @@ static int device_is_shearwater(dc_user_device_t *device)
 	return !strcmp(device->vendor, "Shearwater");
 }
 
+static int device_is_hw(dc_user_device_t *device)
+{
+	return !strcmp(device->vendor, "Heinrichs Weikamp");
+}
```

Comment:
same_string() is what we use - if for some reason you get a vendor=null your code will crash

and I think maybe this would be happier as a macro?

#define IS_HW(_d) same_string((_d)->vendor, "Heinrichs Weikamp")